app/views/issues/_form.html.erb

```diff
@@ -6,12 +6,11 @@
       label: false,
       input_html: {
         class: 'textile',
-          data: {
-            behavior: 'auto-save',
-            autosave_key: "#{current_project.id}-issue-#{ @issue.persisted? ? "edit-#{@issue.id}" : 'new' }",
-            help_url: markup_path,
-            preview_url: preview_path
-          }
+        data: textile_form_urls.merge({
+          autosave_key: "#{current_project.id}-issue-#{ @issue.persisted? ? "edit-#{@issue.id}" : 'new' }",
+          behavior: 'auto-save',
+          content: @issue.text
```

Comment:
I don't think we need to append a `content` data attribute to a textarea. We can always reference the text area value. Managing the value in two places means there's no source of truth. We should trust the value of value :D